src/frontends/pytorch/src/op/log.cpp

```diff
@@ -0,0 +1,40 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "openvino/frontend/pytorch/node_context.hpp"
+#include "openvino/opsets/opset10.hpp"
+#include "utils.hpp"
+
+namespace ov {
+namespace frontend {
+namespace pytorch {
+namespace op {
+
+OutputVector translate_log(NodeContext& context) {
+    auto x = context.get_input(0);
```

Comment:
please add a comment that it has a deal with natural logarithm as I know

Reply:
added